homeassistant/components/scene/hunterdouglas_powerview.py

```diff
@@ -68,7 +68,8 @@ def name(self):
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        return {"roomName": self.scene_data["roomName"]}
+        return {"roomName": self.scene_data["roomName"],
+                "icon": "mdi:blinds"}
```

Comment:
@balloob you mean adding the icon property to the scenario according to the following documentation ? :

https://home-assistant.io/getting-started/customizing-devices/

If so, that is what I am trying to avoid. I read scenes from the PowerView hub and import them into hass. I don't want to have to go through a list of all these scenes and have to customize the icon for any of these.

In a sense I want to be able to override domain icon of scene with another one.